src/jumio-api/jumio-api.service.ts

```diff
@@ -100,6 +107,98 @@ export class JumioApiService {
     return response.data;
   }
 
+  async uploadScreeningData(
+    url: string,
+    token: string,
+    data: ScreeningData,
+  ): Promise<JumioStandaloneUpload> {
+    const jsonData = JSON.stringify(data);
+    const config = {
+      method: 'post',
+      url: url,
+      headers: {
+        'User-Agent': 'IronFish Website/v1.0',
+        'Content-Type': 'application/json',
+        Authorization: `Bearer ${token}`,
+      },
+      data: jsonData,
+    };
+    const promise = axios(config);
+
+    const response = await promise.catch((error) => {
+      if (axios.isAxiosError(error)) {
+        this.logger.warn(`${JSON.stringify(error.response?.data)}`);
+        throw new BadRequestException(`Error updating standalone ${url}`);
+      }
+      throw error;
+    });
+
+    return response.data as JumioStandaloneUpload;
+  }
+
+  async putStandaloneScreening(
+    url: string,
+    token: string,
+  ): Promise<JumioPutStandaloneScreening> {
+    const config = {
+      method: 'put',
+      url: url,
+      headers: {
+        'User-Agent': 'IronFish Website/v1.0',
+        'Content-Type': 'application/json',
+        Authorization: `Bearer ${token}`,
+      },
+    };
+    const promise = axios(config);
+
+    const response = await promise.catch((error) => {
+      if (axios.isAxiosError(error)) {
+        this.logger.warn(`${JSON.stringify(error.response?.data)}`);
+
+        throw new BadRequestException(`Error updating standalone ${url}`);
+      }
+      throw error;
+    });
+
+    return response.data as JumioPutStandaloneScreening;
+  }
```

Comment:
If I used the code below (`axios.put<T>`) instead of the `axios(config)` method I used above, I was getting an error for invalid token when doing integration testing, not sure why.
```
{"level":40,"time":1678503280868,"pid":34131,"hostname":"Iron-Fish-MBP.local","msg":"{\"type\":\"about:blank\",\"title\":\"Unauthorized: wrong authorization token\",\"status\":401,\"detail\":\"[]\",\"instance\":\"/error\"}"}
```
```
  async putStandaloneScreening(
    url: string,
    token: string,
  ): Promise<JumioPutStandaloneScreening> {
    const config = {
      method: 'put',
      url: url,
      headers: {
        'User-Agent': 'IronFish Website/v1.0',
        'Content-Type': 'application/json',
        Authorization: `Bearer ${token}`,
      },
    };
    const promise = axios(config);

    const response = await promise.catch((error) => {
      if (axios.isAxiosError(error)) {
        this.logger.warn(`${JSON.stringify(error.response?.data)}`);

        throw new BadRequestException(`Error updating standalone ${url}`);
      }
      throw error;
    });

    return response.data as JumioPutStandaloneScreening;
  }
  
```

```
  async putStandaloneScreening(
    url: string,
    token: string,
  ): Promise<JumioPutStandaloneScreening> {
    const config = {
      headers: {
        'User-Agent': 'IronFish Website/v1.0',
        'Content-Type': 'application/json',
        Authorization: `Bearer ${token}`,
      },
    };
    const promise = axios.put<JumioPutStandaloneScreening>(url, config);

    const response = await promise.catch((error) => {
      if (axios.isAxiosError(error)) {
        this.logger.warn(`${JSON.stringify(error.response?.data)}`);

        throw new BadRequestException(`Error updating standalone ${url}`);
      }
      throw error;
    });

    return response.data;
}
```